packages/block-editor/src/layouts/grid.js

```diff
@@ -351,7 +393,7 @@ function GridLayoutTypeControl( { layout, onChange } ) {
 		minimumColumnWidth || '12rem'
 	);
 
-	const isManual = !! columnCount ? 'manual' : 'auto';
+	const isManual = !! minimumColumnWidth ? 'auto' : 'manual';
```

Comment:
Nit: Since this isn't a boolean, `isManual` feels a bit misleading. Maybe something like `columnMode`? Names are hard :smile: